04.ls/ls_command.rb

```diff
@@ -43,4 +67,38 @@ def display_files(files)
   end
 end
 
+def display_file_details(files)
+  file_stats = files.to_h { |file| [file, File::Stat.new(file)] }
+
+  puts "total #{file_stats.values.sum(&:blocks)}"
+  print find_file_details(file_stats).join("\n")
```

Comment:
findは見つけ出す、という意味なので、この処理自体はどこかから何かを見つけてくる処理とは考えにくいですね。
詳細情報の行を作る、みたいな意味で

```suggestion
  print build_file_detail_rows(file_stats).join("\n")
```

のようにするのはどうでしょうか？（ちょっと長いですが）